week2/README.md

```diff
@@ -12,4 +14,4 @@ Through these new concepts a component will be able to update information within
 
 **Homework:**
 
-Create a video player to server as practice for updating sibling props
+Create a video player to  server as practice for updating sibling props

```

Comment:
I put a double space to be able to comment here this time.

What do you mean by `video player to server`?

Reply:
Oops that's supposed to be 'video player to *serve* as practice' not server